Makefile

```diff
@@ -283,7 +283,7 @@ kind-up kind-down gardener-up gardener-down register-local-env tear-down-local-e
 
 kind2-up kind2-down gardenlet-kind2-up gardenlet-kind2-down: export KUBECONFIG = $(GARDENER_LOCAL2_KUBECONFIG)
 
-kind-ha-up register-kind-ha-single-zone-env tear-down-kind-ha-single-zone-env register-kind-ha-multi-zone-env tear-down-kind-ha-multi-zone-env ci-e2e-kind-ha-single-zone ci-e2e-kind-ha-multi-zone: export KUBECONFIG = $(GARDENER_LOCAL_HA_KUBECONFIG)
+kind-ha-up kind-ha-down gardener-ha-up register-kind-ha-single-zone-env tear-down-kind-ha-single-zone-env register-kind-ha-multi-zone-env tear-down-kind-ha-multi-zone-env ci-e2e-kind-ha-single-zone ci-e2e-kind-ha-multi-zone: export KUBECONFIG = $(GARDENER_LOCAL_HA_KUBECONFIG)
```

Comment:
@rfranzke We have to make same changes for local `env` setup right ?
register-kind-ha-env instead of ` register-kind-ha-single-zone-env` `register-kind-ha-multi-zone-env`.

Pls remove these changes also 
https://github.com/gardener/gardener/blob/master/example/provider-local/seed-kind-ha/local-multi-zone/patch-seed.yaml
https://github.com/gardener/gardener/blob/master/example/provider-local/seed-kind-ha/local-single-zone/patch-seed.yaml

Reply:
Yes, thanks, good catches!